lib/cpl.rb

```diff
@@ -201,9 +201,10 @@ def self.process_option_params(params)
 
       @commands_with_extra_options.push(name_for_method.to_sym) if accepts_extra_options
 
-      define_method(name_for_method) do |*provided_args| # rubocop:disable Metrics/MethodLength
+      define_method(name_for_method) do |*provided_args| # rubocop:disable Metrics/BlockLength, Metrics/MethodLength
         if deprecated
-          ::Shell.warn_deprecated("Command '#{command_key}' is deprecated, " \
+          normalized_old_name = ::Helpers.normalize_command_name(command_key)
+          ::Shell.warn_deprecated("Command '#{normalized_old_name}' is deprecated, " \
```

Comment:
Prints, e.g., `cleanup-old-images` instead of `cleanup_old_images` (internal name).